rdt/transformers/numerical.py

```diff
@@ -109,8 +112,7 @@ def _learn_rounding_digits(data):
                     return decimal
 
         # Can't round, not equal after MAX_DECIMALS digits of precision
-        warnings.warn(
-            f"No rounding scheme detected for column '{name}'. Data will not be rounded.")
+        LOGGER.info("No rounding scheme detected for column '%s'. Data will not be rounded.", name)
```

Comment:
We should always use f-strings, the other formats should actually raise lint errors, so it's curious that it does not, the linter probably needs to be tweaked to fix this.